src/Microsoft.Dnx.Loader/LoadContext.cs

```diff
@@ -103,7 +113,11 @@ static LoadContext()
 
         public LoadContext()
         {
-            _contextId = Guid.NewGuid().ToString();
+        }
```

Comment:
Seems that `BuildLoadContext` is the only user of this default constructor and it doesn't add the `LoadContext` itself to the `_contexts`. It feels like way too subtle...